src/cosmoz-tab-card.js

```diff
@@ -60,19 +64,29 @@ const CosmozTabCard = ({ heading }) =>
 		</style>
 
 		<div id="header" part="header">
+			${collapsable
+				? html`<input
+						type="checkbox"
+						@change=${() => setCollapsed((c) => !c)}
+				  />`
+				: nothing}
```

Comment:
I would avoid using a checkbox input. It has it's own internal state. Just put there an icon with a click handler.

Reply:
Put there an iron icon, not the best looking ever but I read a bit of docs after your comment and yes, checkbox is quite bug prone compared to an icon